templates/agenda_item/edit.html.twig

```diff
@@ -0,0 +1,28 @@
+{% extends layout %}
+
+{% trans_default_domain 'agenda_item' %}
+
+{% block title %}{% trans %}Agenda item{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Agenda item{% endtrans %}</h1>
+            <p class="text-muted"><small>{% trans %}Edit agenda item{% endtrans %}</small></p>
+        </div>
+        <div class="col-auto">
+{#            <a name="" id="" class="btn btn-secondary" href="{{ url('agenda_show', {id: agenda.id}) }}" role="button">{% trans %}Cancel{% endtrans %}</a>#}
+        </div>
+    </div>
+{% endblock %}
+
+{% block content %}
+
+    <main role="main" class="col-12 mt-3">
+        {{ form(agenda_item_edit_form) }}
+    </main>
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+{% endblock %}
```

Comment:
Isn't this already inherited from the parent? I mean if we don't have any need to override we probably shouldn't.

Reply:
Removed redundant `parent()` call.